azurelinuxagent/common/logcollector.py

```diff
@@ -195,9 +195,9 @@ def _remove_uncollected_truncated_files(files_to_collect):
 
     @staticmethod
     def _expand_parameters(manifest_data):
-        _LOGGER.info("Using {0} as $LIB_DIR".format(_AGENT_LIB_DIR)) # pylint: disable=W1202
-        _LOGGER.info("Using {0} as $LOG_DIR".format(_EXTENSION_LOG_DIR)) # pylint: disable=W1202
-        _LOGGER.info("Using {0} as $AGENT_LOG".format(_AGENT_LOG)) # pylint: disable=W1202
+        _LOGGER.info("Using %s as $LIB_DIR", _AGENT_LIB_DIR)
```

Comment:
I already cleaned up this file in my pending PR (https://github.com/Azure/WALinuxAgent/pull/1987), sorry you did double work! But thanks for removing this error from the linter conf, I had not done that.